packages/sit-onyx/src/components/OnyxHeadline/OnyxHeadline.vue

```diff
@@ -0,0 +1,66 @@
+<script lang="ts" setup>
+import type { OnyxHeadlineProps } from "./types";
+
+const props = withDefaults(defineProps<OnyxHeadlineProps>(), {
+  monospace: false,
+});
+
+defineSlots<{
+  /**
+   * Headline content.
+   */
+  default(): unknown;
+}>();
+</script>
+
+<template>
+  <component
+    :is="props.is"
+    :class="[
+      'onyx-headline',
+      `onyx-headline--${props.is}`,
+      props.monospace ? 'onyx-headline--monospace' : '',
+    ]"
+  >
+    <slot></slot>
+  </component>
+</template>
+
+<style lang="scss">
+// use :where for styles that should be able to be easily overridden by the user
+:where(.onyx-headline) {
+  margin: 0; // remove default browser margins
+}
+
+.onyx-headline {
+  font-weight: 600;
+  font-family: var(--onyx-font-family);
+  color: var(--onyx-color-text-neutral-intense);
```

Comment:
should we put the color to :where too? in the past there were situations where headlines were placed on an inverted background (e.g. dark/primary in lightmode), so it might be better if overriding is proven easy :)